.dockerignore

```diff
@@ -1,3 +1,6 @@
 frontend/node_modules
 backend/server/dist
 bytebase-build
+scripts/Dockerfile*
```

Comment:
no need to change this file.

Reply:
take look old version of Dockerfile line 86 `COPY ./scripts /usr/local/bin/` adding this ignore voids unnecessary files, I am keeping it in case others may doing similar act.

Reply:
scripts/Dockerfile does matter and we should not ignore.

Reply:
the syntax is `Dockerfile*` as there are other Dockerfile under this dir., personally I would suggest separate `scirpts` and `Dockerfile(s)`